.github/workflows/rust-apply-patch.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
This file is very similar to `docker-apply-patch.sh`, and I assume there will be more `$package-apply-patch.sh` coming... It seems it should be possible to just have a base script that receives a couple of parameters for each package, no?

Reply:
All right.
I will then try to create `common-apply-patch.sh` that includes common parts.
And make other scripts include the common script.

Reply:
Done.
Split out common parts to `common.sh`.

Reply:
Thanks, this looks much better!